README.md

```diff
@@ -0,0 +1,14 @@
+# Media Consent Form
+
+This is the form for students to grant media consent for our events, so we can take pictures and record video at will.
+
+
+## Making a form
+
+You must have `pdflatex`, `make`, and either `svg2pdf` or `inkscape` installed.
```

Comment:
you could also suggest a normal GUI tex editor ??? :) Like TeXStudio or TeXMaker ????

Reply:
I'm not sure it's compatible. The build process builds SVGs into PDF assets, which I doubt those editors will do automatically. Whilst I don't like this custom build process, it works, well, and is stable.